server.js

```diff
@@ -21,27 +28,30 @@ console.info(`info: Git version: ${gitVersion}`);
 console.info(`info: API address: ${apiUrl}`);
 console.info(`info: IS DEV: ${process.env.IS_DEV}`);
 
+samlEndPoint(app)
 app.use(                       function(req,res,next) { res.setHeader('X-Frame-Options', 'DENY' ); next(); });
 app.use('/app/authorize.html', function(req,res,next) { res.setHeader('X-Frame-Options', 'ALLOW'); next(); });
 
 // Configure static files to serve
 app.use('/favicon.png',   express.static(__dirname + '/app/images/favicon.png', { maxAge: 24*60*60*1000 }));
 app.use('/app',           express.static(__dirname + '/app'     , { setHeaders: setCustomCacheControl }));
 app.use('/app/libs',      express.static(__dirname + '/node_modules/@bower_components', { setHeaders: setCustomCacheControl }));
-app.all('/app/*',         function(req, res) { res.status(404).send(); } );
+//app.all('/app/*',         function(req, res) { res.status(404).send(); } );
```

Comment:
why this o the one at line 54 has been comented?

Reply:
left behind in error, my bad